Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -768,6 +776,49 @@ namespace AzToolsFramework
         return lastDescendant;
     }
 
+    AZ::Dom::Path DPERowWidget::BuildDomPath()
+    {
+        auto pathToRoot = GetDPE()->GetPathToRoot(this);
+        AZ::Dom::Path rowPath = AZ::Dom::Path();
+
+        auto reverseIndexIter = pathToRoot.rbegin();
+        for (reverseIndexIter; reverseIndexIter != pathToRoot.rend(); ++reverseIndexIter)
+        {
+            rowPath.Push(*reverseIndexIter);
+        }
```

Comment:
FYI: This can use the C++20 https://en.cppreference.com/w/cpp/ranges/reverse_view to access the elements using a ranged-for loop.

The `<AzCore/std/ranges/reverse_view.h>` header contains the functionality.

```suggestion
        for (auto reversePathEntry : pathToRoot | AZStd::views::reverse)
        {
            rowPath.Push(reversePathEntry);
        }
```

Reply:
Done. This is cool, but that bar character feels out of place. I think I can get used to it though.